2021_CarlhoffNature/POSEIDON.yml

```diff
@@ -0,0 +1,26 @@
+poseidonVersion: 2.7.1
+title: 2021_CarlhoffNature
+description: Genome of a Middle Holocene hunter-gatherer from Wallacea
+contributor:
+- name: Thiseas C. Lamnidis
+  email: thiseas_christos_lamnidis@eva.mpg.de
+  orcid: 0000-0003-4485-8570
+packageVersion: 0.1.6
```

Comment:
As discussed I think we could set the version number for the first version in the archive to `1.0.0`.